src/documentation/common.md

```diff
@@ -1,9 +1,11 @@
 # Shared specifications for all converters
-This document explains the shared parts among all the converters when converting a 
-data set from a given format into a [NTFS](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md) dataset.
+This document explains the shared parts among all the converters when converting a
+data set from a given format into a [NTFS] dataset.
+
+[NTFS]: https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md
 
 ## Data prefix
-The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique 
+The construction of NTFS objects IDs requires, for uniqueness purpose, that a unique
```

Comment:
Interesting, I tend to let trailing spaces when it is a paragraph (to mark the fact that it is not the end of the line, only a wrapping for 80 characters)...

Reply:
vscode is removing it, and git (git-gui, github) tend to complain about that, so I removed this.
(note that I don't print whitechars in my editor as I've seen you do :wink: )

Reply:
generally it's not recommended to remove trailing spaces in mardown.

Reply:
After discussion, stayed on <br> as it seems to be the only use of trailing-whitespaces so far in md.